tests/components/wemo/test_light_bridge.py

```diff
@@ -1,16 +1,19 @@
 """Tests for the Wemo light entity via the bridge."""
```

Comment:
It would be better to keep all the light platform tests in the same module. This can be addressed in the future.